ui/shared/components/panel/family/Family.jsx

```diff
@@ -89,6 +89,13 @@ const FAMILY_FIELD_RENDER_LOOKUP = {
       <AnalysedBy analysedByList={analysedByList} compact={compact} familyGuid={familyGuid} />
     ),
   },
+  [FAMILY_FIELD_EXTERNAL_DATA]: {
+    internal: true,
+    component: TagFieldView,
+    tagOptions: FAMILY_EXTERNAL_DATA_OPTIONS,
+    simplifiedValue: true,
```

Comment:
I see what `simplifiedValue` controls in `TagFieldView` but is there a conceptual rule for which fields get simplifiedValue set to true?

Reply:
So a rough rule is if your field has a value which is just a list of strings it probably needs the simplified view, whereas if your field has a values thats a list of objects (like variant tags) it need to more complex handling